lib/Log/Any/Adapter/DERIV.pm

```diff
@@ -265,45 +295,45 @@ sub format_line {
     local $Term::ANSIColor::EACHLINE = "\n";
     my ($ts, $level) = splice @details, 0, 2;
     $from = shift @details;
-    return join ' ',
-        colored(
-            $ts,
-            qw(bright_blue),
-        ),
-        colored(
-            $level,
-            @colours,
-        ),
-        colored(
-            $from,
-            qw(grey10)
-        ),
-        map {
-            colored(
-                $_,
-                @colours,
-            ),
-        } @details;
+
+    return join ' ', colored($ts, qw(bright_blue),), colored($level, @colours,), colored($from, qw(grey10)), map { colored($_, @colours,), } @details;
```

Comment:
if we're going to squash all this onto one line, I'd also suggest dropping that trailing comma, it looks like a mistake